app/src/pages/CollectionPage/CollectionPage.tsx

```diff
@@ -29,23 +29,27 @@ export const CollectionPage = (): React.ReactElement => {
         <Text variant='error'>Collection failed to load</Text>
       ) : (
         <Stack direction={Direction.Vertical} isFullWidth={true} isFullHeight={true} childAlignment={Alignment.Center} contentAlignment={Alignment.Start}>
-          {collection.bannerImageUrl && (
-            <Box variant='card' isFullWidth={true}>
-              <Image source={collection.bannerImageUrl} alternativeText='image' isFullWidth={true} fitType='cover' />
-            </Box>
-          )}
+          <Box height='200px'>
+            <LayerContainer>
+              <LayerContainer.Layer isFullHeight={false} alignmentVertical={Alignment.Start}>
+                {collection.bannerImageUrl && (
+                  <Box variant='cpBanner' height='250px' isFullWidth={true}>
```

Comment:
instead of adding the padding to the variant could you try use a PaddingView. it should work like:
```
<PaddingView paddingBottom='...'>
  <Box ...>
    ...
  </Box>
</PaddingView>
```

Reply:
the idea is we should only be controlling overview things in the theme not application layout stuff which is what this is doing